python/plugins/processing/algs/help/qgis.yaml

```diff
@@ -0,0 +1,442 @@
+
+qgis:randompointsinsidepolygonsvariable: >
+  This algorithm creates a new point layer with random points insides the ppolygons of a given layer. The number of points in each polygon can be defined as a fixed count or as a density value. The count/density valu is taken from an attribute, so it can be different for each polygons in the input layer.
+
+qgis:randomselection: >
+  This algorithm takes a vector layer and selects a subset of its features. No new layer is generated by this algorithm.
+
+  The subset is defined randomly,  using a percentage or count value to define the total number of features in the subset.
+
+qgis:randomselectionwithinsubsets: >
+  This algorithm takes a vector layer and selects a subset of its features. No new layer is generated by this algorithm.
+
+  The subset is defined randomly,  using a percentage or count value to define the total number of features in the subset. 
+
+  The percentage/count value is not applied to the whole layer, but instead to each category. Categories are defined according to a given attribute, which is also specified as an input parameter for the algorithm.
+
+qgis:rasterlayerhistogram: >
+  This algorithm generates a histogram with the values of a raster layer.
+
+  The raster layer must have a single band.
+
+qgis:rasterlayerstatistics: >
+  This algorithm computes basic statistics from the values in a raster layer.
+
+  The raster layer must have a single band.
+
+qgis:refactorfields: >
+  This algorithms allows editing the structure of the attributes table of a vector layer. Fields can be modified in their type and name, using a fields mapping
+
+  The original layer is not modified. A new layer is generated, which contains a modified attributes table, accordint to the provided fields mapping
+
+qgis:regularpoints:
+
+
+qgis:reprojectlayer: >
+  This algorithm reprojects a vector layer. It creates a new layer with the same features as the input one, but with geometries reprojected to a new CRS.
+
+  Attributes are not modified by this algorithm.
+
+qgis:saveselectedfeatures: >
+  This algorithm creates a new layer with all the selected features in a given vector layer. 
+
+  If the selected layer has no selected features, all features will be added to the resulting feature.
+
+qgis:selectbyattribute: >
+  This algorithms creates a selection in a vector layer. The criteria for selected features is defined based on the values of an attribute from the input layer.
+
+qgis:selectbyattributesum:
+
+
+qgis:selectbyexpression: >
+  This algorithms creates a selection in a vector layer. The criteria for selecting features is based on a QGIS expression.
+
+  For more information about expressions see <a href ="https://docs.qgis.org/2.8/en/docs/user_manual/working_with_vector/expression.html">here</a>
```

Comment:
@m-kuhn I don't know how to link to the most recent version...Actually, the https://docs.qgis.org/en/docs url redirects to  https://docs.qgis.org/2.8/en/docs...but  https://docs.qgis.org/en/docs/user_manual/working_with_vector/expression.html returns a 404